client/account-recovery/lost-password/style.scss

```diff
@@ -0,0 +1,24 @@
+.lost-password__title {
+	font-size: 1.5em;
+	font-weight: bold;
+}
+
+.lost-password__user-login-input {
+	//Use a more specific selector so we don't have to use !important to overide any css
+	&.form-text-input {
```

Comment:
Why not merge this into the parent selector if it's the only styles being applied here?

```css
.lost-password__user-login-input.form-text-input {
```